### PR TITLE
GiniCapture: added popup with error message for unsupported QR code format (PIA-1563)

### PR DESCRIPTION
**Pull request by zladzeyka**
GiniCapture:
  *  added popup with error message for unsupported QR code format (PIA-1563)
GiniConfiguration:
  * added configuration for unsupported popup: background color, text color and button color (PIA-1563)

### Description

added popup for unsupported qr code

### Merging
Automatic

### Issues
https://ginis.atlassian.net/browse/PIA-1563

### CODE REVIEW

File: GiniCapture/Classes/Core/Screens/Camera/CameraViewController.swift [hunk 1]
```diff
@@ -236,6 +236,27 @@ extension CameraViewController {
         self.toolTipView?.alpha = 0
     }
     
+    /**
```
Comment (a-szotyori) [inline GiniCapture/Classes/Core/Screens/Camera/CameraViewController.swift]: This sneaked in from https://github.com/gini/gini-vision-lib-ios/tree/PIA-711 and causes a build error.

**Review by a-szotyori — CHANGES_REQUESTED** [review]
Just some code needs to be removed.

File: GiniCapture/Classes/Core/Screens/Camera/CameraViewController.swift (hunk 1 above)
Comment (a-szotyori) [reply GiniCapture/Classes/Core/Screens/Camera/CameraViewController.swift]: If I comment out this unused code, then it builds and works fine.

Comment (zladzeyka) [reply GiniCapture/Classes/Core/Screens/Camera/CameraViewController.swift]: I need to practice more patches )

**Review by a-szotyori — APPROVED** [review]

**Pull request merged**